std/math.d

```diff
@@ -5411,6 +5412,7 @@ if (isFloatingPoint!(X))
  *  $(D true) if $(D_PARAM x) is finite.
  */
 bool isFinite(X)(X x) @trusted pure nothrow @nogc
+    if (isFloatingPoint!X)
```

Comment:
If constraints need to be indented on the same level as the declaration: https://dlang.org/dstyle.html#phobos_declarations